CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.16.1 - 2019-10-16
+### Fixed
+- Ignore deploySubPath task for local git deploys #1185
```

Comment:
Might as well copy directly from the rel branch changelog, which has actual links: https://github.com/microsoft/vscode-azureappservice/blob/rel/0.16.1/CHANGELOG.md